Microsoft.Azure.CosmosRepository/src/Internals/InternalPatchOperation.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using Microsoft.Azure.Cosmos;
+
+namespace Microsoft.Azure.CosmosRepository.Internals
+{
+    internal class InternalPatchOperation
+    {
+        public PatchOperationType Type { get; }
+        public PropertyInfo PropertyInfo { get; }
+
+        public object NewValue { get; }
```

Comment:
I'd prefer if we could somehow still flow the actual generic type `T`, instead of `object` here. I see that their patch operation class has `T`.

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/src/Patch/PatchOperation.cs

Reply:
This object is purely for the implementation of the in-memory implementation the `InternalPatchOperation` class is not used for actual implementation.